rnote-ui/src/iconpicker.rs

```diff
@@ -195,30 +265,33 @@ impl IconPicker {
 
         // Factory
         let icon_factory = SignalListItemFactory::new();
-        icon_factory.connect_setup(move |_factory, list_item| {
+        icon_factory.connect_bind(move |_factory, list_item| {
```

Comment:
Was there a specific issue / reason that this is changed?

Reply:
Yes, when connecting to the setup signal, `list_item.item()` is always `None`. This doesn't happen when you use bind (`ChangePenStyleListFactory` uses this too BTW).

Reply:
Hm, I was using expressions here which are supposed to be in `setup()`, I believe that was correct. (see: https://gtk-rs.org/gtk4-rs/stable/latest/book/list_widgets.html ) Not sure why you replaced that. But It doesn't matter when this is replaced by flowbox

Reply:
You could have used an expression as well, that is usually how it should be done when using models with the *View list widgets, because the models don't map one to one to the widgets, they get recycled. One does get away with it when the model is never modified or reordered though.

Reply:
Alright, thanks for the info.